languages/java/config.json

```
@@ -28,6 +28,12 @@
         "concepts": ["booleans"],
         "prerequisites": ["basics"]
       },
+      {
+        "slug": "lists",
+        "uuid": "e9e2491b-130e-4358-a54d-2c7d2e3e527c",
+        "concepts": ["generic-types", "lists"],
+        "prerequisites": ["arrays", "foreach-loops", "interface"]
```

I think `interface` should probably be `interfaces` (plural):

```suggestion
        "prerequisites": ["arrays", "foreach-loops", "interfaces"]
```